lib/reporters/base.js

```diff
@@ -182,9 +182,11 @@ function stringifyDiffObjs(err) {
  * @return {string} Diff
  */
 var generateDiff = (exports.generateDiff = function(actual, expected) {
+  var err = {actual: actual, expected: expected};
```

Comment:
Is this line IE11 compatible?

Reply:
It should be, as according to [this chart](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#Browser_compatibility) IE supports object initializers.

Reply:
Please revert your changes in this function and insert `try/catch`.
An exception in generating diff is not a fatal error, so I propose to catch/continue and just print a message. For example:
```js
try {
    return exports.inlineDiffs
      ? inlineDiff(actual, expected)
      : unifiedDiff(actual, expected);
  } catch (err) {
    var msg =
      '\n      ' +
      colorLines('diff added', '+ expected') +
      ' ' +
      colorLines('diff removed', '- actual:  failed to generate Mocha diff') +
      '\n';
    return msg;
  }
```